exercises/scale-generator/description.md

```diff
@@ -1,7 +1,7 @@
-Write a program that can generate musical scales. It should accept
-a tonic, or starting note, and a set of intervals. It should be able to
-output the notes in the given scale, starting with the tonic and
-following the specified interval pattern.
+Generate musical scales accepting a tonic, or starting note, and
+a set of intervals. It should be able to output the notes in the
```

Comment:
Previously, "it" referred to the program, and now I'm not sure "it" makes sense any longer. Maybe it will make sense by simply removing the words "It should"?